### PR TITLE
Loop when player is running

### PR DESCRIPTION
**Pull request by SkinyMonkey**
I need the loop option to be activable even if the player is already playing.

I updated demo accordingly to test if you want to check it.

Thank you for this great component by the way!

### CODE REVIEW

File: src/demo/App.js [hunk 1]
```diff
@@ -132,7 +137,7 @@ export default class App extends Component {
               onPause={this.onPause}
               onBuffer={() => console.log('onBuffer')}
               onSeek={e => console.log('onSeek', e)}
-              onEnded={() => this.setState({ playing: false })}
+              onEnded={() => this.setState({ playing: loop ? true : false })}
```
Comment (SkinyMonkey) [inline src/demo/App.js]: Useless ternary, 'don't know what I was thinking about.

**Comment by CookPete** [comment]
I like the idea, but don't forget to run [`npm run lint`](https://github.com/CookPete/react-player/blob/master/CONTRIBUTING.md#linting) before commiting.

**Comment by SkinyMonkey** [comment]
Sorry for that, It should be fixed now.

**Comment by CookPete** [comment]
Nice work 👍

**Pull request merged**